spec/acceptance/helpers/repo_helper.rb

```diff
@@ -0,0 +1,211 @@
+module Acceptance
+  module Helpers
+    module RepoHelper
+
+      # Install a yum repo
+      #
+      # +host+: Host object on which the yum repo will be installed
+      # +repo_filename+: Path of the repo file to be installed
+      #
+      # @fails if the specified repo file cannot be installed on host
+      def copy_repo(host, repo_filename, repo_name = 'simp_manual.repo')
+        if File.exists?(repo_filename)
+          puts('='*72)
+          puts("Using repos defined in #{repo_filename}")
+          puts('='*72)
+          scp_to(hosts, repo_filename, "/etc/yum.repos.d/#{repo_name}")
+        else
+          fail("File #{repo_filename} could not be found")
+        end
+      end
+
+      def install_puppet_repo(host)
+        case ENV['BEAKER_puppet_repo']
+        when 'true'
+           install_repo = true
+        when 'false'
+           install_repo = false
+        else
+           install_repo = true
+        end
+
+        if install_repo
+          puppet_collection = ENV['PUPPET_COLLECTION'] || 'puppet5'
+
+          puts('='*72)
+          puts("Using Puppet #{puppet_collection} repo from yum.puppetlabs.com")
+          puts('='*72)
+
+          if host.host_hash[:platform] =~ /el-7/
+            family = 'el-7'
+          elsif host.host_hash[:platform] =~ /el-6/
+            family = 'el-6'
+          else
+            fail("install_puppet_repo(): Cannot determine puppet repo for #{host.name}")
+          end
+          url = "http://yum.puppetlabs.com/#{puppet_collection}/#{puppet_collection}-release-#{family}.noarch.rpm"
+          on(host, "yum install #{url} -y")
+        end
+      end
+
+      # Install a SIMP packagecloud yum repo
+      #
+      # - Each repo is modeled after what appears in simp-doc
+      # - See https://packagecloud.io/simp-project/ for the reponame key
+      #
+      # +host+: Host object on which SIMP repo(s) will be installed
+      # +reponame+: The base name of the repo, e.g. '6_X'
+      # +type+: Which repo to install:
+      #   :main for the main repo containing SIMP puppet modules
+      #   :deps for the SIMP dependency repo containing OS or application
+      #         RPMs not available from standard CentOS repos
+      #
+      # @fails if the specified repo cannot be installed on host
+      def install_internet_simp_repo(host, reponame, type)
+        case type
+        when :main
+          full_reponame = reponame
+          # FIXME: Use a gpgkey list appropriate for more than 6_X
+          repo = <<~EOM
```

Comment:
Should we pull these from fixtures?

Reply:
We need to figure out how to pull from simp-doc, because we really want to test what we tell users.  Another ticket?

Reply:
I'm pretty sure we have an Epic that covers this, so I'm good.